pytorch_lightning/strategies/deepspeed.py

```diff
@@ -334,17 +425,39 @@ def _load_config(self, config):
                 config = json.load(f)
         return config
 
-    def setup_distributed(self):
-        reset_seed()
+    def set_world_ranks(self) -> None:
```

Comment:
same comment as elsewhere, we can make this protected

Reply:
@ananthsub i don't think we can because Accelerator Connector calls it in the _lazy_init_strategy.

Reply:
ah I didn't realize that!